runhouse/servers/cluster_servlet.py

```diff
@@ -280,6 +256,13 @@ async def aperiodic_status_check(self):
                 )
                 await asyncio.sleep(INCREASED_STATUS_CHECK_INTERVAL)
             else:
+                # make sure that the thread will go to sleep, even if the interval size == -1
```

Comment:
this is because we're reusing this thread for autostop too, right?

Reply:
yeah, exactly @rohinb2